ixmp/backend/base.py

```diff
@@ -902,6 +904,20 @@ def set_meta(self, s: Scenario, name, value):
             If *value* is not a valid type.
         """
 
+    @abstractmethod
+    def delete_meta(self, s, name_or_names):
```

Comment:
no need to call this arg `name_or_names`, `name` is sufficiently clear

Reply:
updated